backend/maelstro/metadata/meta.py

```diff
@@ -79,7 +79,35 @@ def get_gslayer_from_gn_link(
             workspace_name=ows_url.lstrip("/").split("/")[0], layer_name=layer_name
         )
 
-    def update_geoverver_urls(self, mapping: dict[str, list[str]]) -> tuple[str, str]:
+    def _apply_xslt(self, xslt_path: str) -> bytes:
+        with open(xslt_path, encoding="utf8") as xslt_file:
+            xslt = etree.parse(xslt_file)
+        transform = etree.XSLT(xslt)
+        xml_root = etree.parse(BytesIO(self.xml_bytes))
+        return self.dump_tree_to_bytes(transform(xml_root))
+
+    def apply_xslt(self, xslt_path: str) -> tuple[str, str]:
```

Comment:
This seems to return a string. Seems only used in tests.

Reply:
no, it is called by the child class MetaZip via super()
https://github.com/georchestra/maelstro/blob/GSREN-237_xsl_transform/backend/maelstro/metadata/meta.py#L186